### PR TITLE
chore(deps): update express COMPASS-8293

### PR DESCRIPTION
**Pull request by gribnoysup**
Update `express` to update `body-parser` to update `send` to resolve autogenerated vuln tickets COMPASS-8293 through COMPASS-8297

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,8 +94,5 @@
     "packages/*",
     "configs/*",
     "scripts"
-  ],
-  "overrides": {
-    "mongodb-client-encryption": "^6.1.0"
```
Comment (gribnoysup) [inline package.json:99]: Drive-by, this was not doing anything anymore

File: package-lock.json
Comment (gribnoysup) [inline package-lock.json:1]: `express` updated with `npm update`, so only lockfile got changed

**Review by mabaasit — APPROVED** [review]

File: package-lock.json
Comment (Anemy) [reply package-lock.json:1]: Looks like this causes an error with a fresh install + boostrap + npm start on main:
```
[webpack-cli] Error: Cannot find module 'webpack/hot/dev-server'
Require stack:
- /Users/rhys/mongodb/compass/node_modules/webpack-dev-server/lib/Server.js
```
Maybe the versions got mixed up.
